scripts/run_mypy_checks.py

```diff
@@ -33,6 +33,7 @@
     'proto_files/',
     'scripts/linters/test_files/',
     'third_party/',
+    'venv/'
```

Comment:
Why is this added?

Reply:
@vojtechjelinek see [this gitter thread](https://gitter.im/oppia/oppia-chat?at=61c166a43c203022427abaa8). bhavukJain found that mypy was trying to type-check their virtual environment under `venv/`

Reply:
Ok